src/components/layouts/MobileMenu.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Box,
+  Flex,
+  Modal,
+  ModalContent,
+  ModalOverlay,
+  useDisclosure
+} from '@chakra-ui/react'
+import { CloseIcon } from '@chakra-ui/icons';
+
+import { HamburgerIcon } from '../UI/icons';
+import { Search } from '../UI/search';
+import { HeaderButtons } from '../UI'
+
+import { BORDER_WIDTH } from '../../constants';
+
+export const MobileMenu: React.FC = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+
+  return (
+    <>
+      {/* HAMBURGER MENU ICON */}
+      <Box
+        as='button'
+        p={4}
+        display={{ base: 'block', md: 'none' }}
+        color='primary'
+        _hover={{ bg: 'primary', color: 'bg' }}
+        onClick={onOpen}
+      >
+        <HamburgerIcon />
+      </Box>
+
+      {/* MODAL */}
+      <Modal isOpen={isOpen} onClose={onClose} motionPreset='none'>
+
+        <ModalOverlay />
+        <ModalContent>
+          {/* MOBILE MENU */}
+          <Flex
```

Comment:
container reaches the botto of the screen and [should have bottom padding too](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=885%3A8258&t=Ic6WpLw62Cylvxti-0)

<img width="376" alt="Screen Shot 2022-11-23 at 12 37 52" src="https://user-images.githubusercontent.com/948922/203587475-f4664445-fff0-41e6-b405-e629cc8116fe.png">